airbyte/mcp/connector_development.py

```
@@ -0,0 +1,357 @@
+            if not stream:
+                return f"Stream '{stream_name}' not found in source streams"
+
+            records, logs, slices_processed = _read_stream_records(stream, max_records)
+
+            return _generate_stream_summary(stream_name, records, logs, slices_processed)
+
+        except Exception as e:
+            return f"Error testing stream read: {e!s}"
+
+    @app.tool()
+    def get_manifest_schema(format: str = "yaml") -> str:
```

Ruff (0.11.9)

Function argument `format` is shadowing a Python builtin

(A002)